File: NIV BÁSICO/C++/ContarPalabras/Solucion.cpp
#include<iostream>
#include<string>
using namespace std;
int ContarEspacios(string frase){
    int contador=0;
    for(char c:frase){
        if(c==' '){
            contador++;
        }
    }
    return contador;
}
int main(){
    string frase;
    cout<<"-----------------------------------------\n";
    cout<<"Ingrese una frase: "; getline(cin,frase);
    cout<<"-----------------------------------------\n";
    cout<<"Total de Palabras: "<<ContarEspacios(frase)+1;
    return 0;
}

File: NIV BÁSICO/Python/ContarPalabras/Solucion.py
def ContarEspacios(frase):
    contador=0
    for c in frase:
        if c==' ':
            contador+=1
    return contador
def main():
    print("------------------------")
    frase=input("Ingrese una frase: ")
    print("-------------------------")
    print("Total de Palabras: ", ContarEspacios(frase)+1)
main()